lib/Interpreter/CIFactory.cpp

```diff
@@ -203,6 +203,13 @@ namespace {
       }
 #endif
 
+      // Ignore warnings about redeclaration with/without dllexport & dllimport
+      sArguments.addArgument("-Wno-dll-attribute-on-redeclaration");
+      sArguments.addArgument("-Wno-inconsistent-dllimport");
```

Comment:
"// Windows headers add "__declspec(dllexport) __cdecl" to routines such as printf,
 // causing a lot of warnings for different redeclarations (eg. coming from the test suite).
 // Do not warn about such cases."